build.sbt

```diff
@@ -1692,6 +1691,8 @@ def findJar(files: Seq[Attributed[File]], dep: ModuleID): Option[Attributed[File
 whitesourceProduct               := "Lightbend Reactive Platform"
 whitesourceAggregateProjectName  := "scala-2.13-stable"
 whitesourceIgnoredScopes         := Vector("test", "scala-tool")
+// for some reason sbt 1.4 issues an unused-setting warning for this, I don't understand why
```

Comment:
There's an sbt/sbt issue for this one: the sbt linter doesn't understand the `.all`/ScopeFilter usage that sbt-whitesource makes use of.

Reply:
@dwijnand I couldn't find the ticket, unless you mean  https://github.com/sbt/sbt/issues/5485 ?